File: VB/MyXtraGrid/Custom Grid Control/CustomizationFormEx.vb
Imports DevExpress.XtraGrid.Views.Grid.Customization
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace MyXtraGrid
    Public Class CustomizationFormEx
        Inherits CustomizationForm

        Public Sub New(ByVal view As DevExpress.XtraGrid.Views.Grid.GridView)
            MyBase.New(view)
        End Sub
        Protected Overrides Function CreateCustomizationListBox() As DevExpress.XtraEditors.Customization.CustomizationListBoxBase
            Return New ColumnCustomizationListBoxEx(Me)
        End Function
    End Class
End Namespace

File: CS/MyXtraGrid/Custom Grid Control/CustomizationFormEx.cs
using DevExpress.XtraGrid.Views.Grid.Customization;
using System;
using System.Collections.Generic;
using System.Linq;

namespace MyXtraGrid {
    public class CustomizationFormEx : CustomizationForm {
        public CustomizationFormEx(DevExpress.XtraGrid.Views.Grid.GridView view)
            : base(view) {
        }
        protected override DevExpress.XtraEditors.Customization.CustomizationListBoxBase CreateCustomizationListBox() {
            return new ColumnCustomizationListBoxEx(this);
        }
    }
}
